components/datetime/src/error.rs

```diff
@@ -25,9 +25,6 @@ pub enum DateTimeError {
     /// An error originating from parsing a pattern.
     #[displaydoc("{0}")]
     Pattern(PatternError),
-    /// An error originating from the [`Write`](std::fmt::Write) trait.
-    #[displaydoc("{0}")]
-    Format(core::fmt::Error),
```

Comment:
Thought: Might be semver breaking

Reply:
I can restore this, I mainly removed this to make sure it's not being used.